packages/cli/lib/services/dryrun.service.ts

```diff
@@ -303,6 +306,10 @@ export class DryRunService {
                 },
                 startedAt: new Date()
             };
+            if (options.saveResponses) {
+                nangoProps.rawSaveOutput = [];
+                nangoProps.rawDeleteOutput = [];
```

Comment:
Is there a time when we do not need to log everything?

Reply:
Yeah, probably but for now we're saving everything for completeness sake